initializr-generator/src/main/java/io/spring/initializr/generator/project/ProjectDescriptionModifications.java

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright 2012-2019 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.spring.initializr.generator.project;
+
+import io.spring.initializr.generator.buildsystem.BuildSystem;
+import io.spring.initializr.generator.version.Version;
+import java.util.Objects;
+import java.util.function.BiConsumer;
+
+public class ProjectDescriptionModifications {
```

Comment:
`ProjectDescriptionDiff` ? (not convinced, just sharing the idea in case that helps us finding something we like)

Reply:
LOL that was the original name I had. You know what they say... "Naming is one of the hardest thing in computer science"